File: EP/vaje7/HelloWorldJava/app/src/main/java/com/example/helloworld/MainActivity.java
package com.example.helloworld;

import androidx.appcompat.app.AppCompatActivity;

import android.os.Bundle;
import android.util.Log;
import android.view.View;
import android.widget.Button;
import android.widget.EditText;
import android.widget.TextView;

import java.util.Date;

public class MainActivity extends AppCompatActivity {

    private final static String TAG = MainActivity.class.getCanonicalName();

    @Override
    protected void onCreate(Bundle savedInstanceState) {
        super.onCreate(savedInstanceState);
        setContentView(R.layout.activity_main);

        final EditText first = findViewById(R.id.firstEt);
        final EditText second = findViewById(R.id.secondEt);
        final Button button = findViewById(R.id.addBtn);
        final TextView textView = findViewById(R.id.resultTv);

        button.setOnClickListener(new View.OnClickListener() {
            @Override
            public void onClick(View v) {
                final int a = Integer.parseInt(first.getText().toString());
                final int b = Integer.parseInt(second.getText().toString());

                final int result = a + b;

                textView.setText(String.valueOf(result));

            }
        });



    }
}

File: EP/vaje7/HelloWorldKotlin/app/src/main/java/ep/helloworld/MainActivity.kt
package ep.helloworld

import androidx.appcompat.app.AppCompatActivity
import android.os.Bundle

import kotlinx.android.synthetic.main.activity_main.*

class MainActivity : AppCompatActivity() {

    override fun onCreate(savedInstanceState: Bundle?) {
        super.onCreate(savedInstanceState)
        setContentView(R.layout.activity_main)

        addBtn.setOnClickListener {
            val a = firstEt.text.toString().toIntOrNull() ?: 0
            val b = secondEt.text.toString().toIntOrNull() ?: 0
            val result = a + b

            resultTv.text = result.toString()
        }


    }
}
